static/js/compose_pm_pill.js

```diff
@@ -9,13 +9,23 @@ exports.initialize_pill = function () {
         container,
         create_item_from_text: user_pill.create_item_from_email,
         get_text_from_item: user_pill.get_email_from_item,
+        onPillCreate: input_pill.onPillCreate,
+        onPillRemove: input_pill.onPillRemove,
```

Comment:
`input_pill.onPillCreate` and `input_pill.onPillRemove` don’t exist. (`input_pill` is the module, not an instance.) What happened here?